docs/network-flow-visibility.md

```diff
@@ -297,11 +326,161 @@ storage size, please modify the `sizeLimit` in the following section.
   name: clickhouse-storage-volume
 ```
 
+To deploy ClickHouse with Persistent Volumes and limited storage size, please refer
+to [Persistent Volumes](#persistent-volumes).
+
 The time interval between the batch commits to the ClickHouse is specified in the
 [Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1)
 as `commitInterval`. The ClickHouse throughput grows sightly when the commit interval
 grows from 1s to 8s. A commit interval larger than 8s provides little improvement on the throughput.
 
+##### Persistent Volumes
```

Comment:
wasn't this already in your PR for persistent volumes?

Reply:
As explained above https://github.com/antrea-io/theia/pull/15#discussion_r870653618, PV is also included in this PR now.

Reply:
Sounds good...